OpenRA.Mods.Common/Pathfinder/PathGraph.cs

```diff
@@ -97,15 +97,15 @@ public PathGraph(CellInfoLayerPool layerPool, MobileInfo mobileInfo, Actor actor
 		{
 			pooledLayer = layerPool.Get();
 			groundInfo = pooledLayer.GetLayer();
+			locomotorInfo = mobileInfo.LocomotorInfo;
```

Comment:
This is the only use of `mobileInfo`, so it would be better if we can pass the `LocomotorInfo` in directly instead.